pkg/daemon/update.go

```diff
@@ -1503,22 +1503,27 @@ func (dn *Daemon) writeUnits(units []ign3types.Unit) error {
 			glog.V(2).Infof("Created symlink unit %q to %s", u.Name, pathDevNull)
 
 			continue
-		}
-
-		if u.Contents != nil && *u.Contents != "" {
+		} else if u.Contents != nil && *u.Contents != "" {
```

Comment:
I thought putting the else here makes this more readable, but not necessary because of the continue

Reply:
No strong feelings here but it does make the `else` condition a bit awkward below, I think